test/e2e/flowaggregator_test.go

```diff
@@ -212,6 +211,11 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 	})
 	require.NoError(t, err, "IPFIX collector did not receive expected number of data records and timed out.")
 
+	if checkBandwidth {
+		// Wait one more export interval of flow aggregator to calculate a more accurate bandwidth value
+		time.Sleep(5 * time.Second)
```

Comment:
Default run time for iperf is 10s and the `collectorCheckTimeout` is 10s, why we want to wait 5 more seconds here when there is possibly no data received during this interval?

Reply:
Yes the run time of iperf and `collectorCheckTimeout` are both 10s, but it's possible that the two records exported by flow aggregator don't cover the whole 10s of the iperf runtime, so I choose to wait for one more record to ensure we could receive all traffic bytes related to iperf.